### PR TITLE
Added Net70 test App

### PR DESCRIPTION
**Pull request by SameerK-MSFT**
Fixes #
Added console app for .net 7.0

**Changes proposed in this request**
New test app

**Testing**
Manual

**Performance impact**
None

**Documentation**
- [ ] All relevant documentation is updated.

### CODE REVIEW

File: tests/devapps/NetCoreTestApp/Program.cs [hunk 1]
```diff
@@ -1,4 +1,7 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿#if Net70App
+#define NET70
```
Comment (pmaytak) [inline tests/devapps/NetCoreTestApp/Program.cs:2]: You don't need to define your own TFM flags; [.NET already includes them](https://github.com/dotnet/designs/blob/main/accepted/2020/net5/net5.md#preprocessor-symbols).

File: tests/devapps/Net70TestApp/Net70TestApp.csproj [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworkNet70>net7.0</TargetFrameworkNet70>
```
Comment (pmaytak) [inline tests/devapps/Net70TestApp/Net70TestApp.csproj:5]: I think it's ok to just add net7.0-windows target to NetCoreTestApp. The core net7 code is the same, -windows suffix just allows to use Windows APIs.

Comment (bgavrilMS) [reply tests/devapps/Net70TestApp/Net70TestApp.csproj:5]: I agree, let's reuse an existing console app. You can even merge the .NET classic and the .NET core console apps if you wish.

Comment (bgavrilMS) [reply tests/devapps/Net70TestApp/Net70TestApp.csproj:5]: You might need to reference net7-windows10.0.17763.0 and not just net7-windows, but do give it a try.

File: tests/devapps/Net70TestApp/Net70TestApp.csproj [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Experimental\" />
+    <Folder Include="Infrastructure\" />
+  </ItemGroup>
+
+</Project>
+
+<!--<Project Sdk="Microsoft.NET.Sdk">
```
Comment (trwalke) [inline tests/devapps/Net70TestApp/Net70TestApp.csproj:36]: Dead code?

File: tests/devapps/NetCoreTestApp/Program.cs [hunk 4]
```diff
@@ -68,13 +77,22 @@ private static string GetAuthority()
...
+                            .WithAuthority(GetAuthority())
+                            .WithLogging(Log, LogLevel.Verbose, true)
+                            .WithExperimentalFeatures();
+#else
             var pcaBuilder = PublicClientApplicationBuilder
                             .Create(s_clientIdForPublicApp)
                             .WithAuthority(GetAuthority())
                             .WithLogging(Log, LogLevel.Verbose, true)
                             .WithExperimentalFeatures()
                             .WithDesktopFeatures();
```
Comment (trwalke) [inline tests/devapps/NetCoreTestApp/Program.cs:92]: nit: you can do 
```CSharp
                  var pcaBuilder = PublicClientApplicationBuilder
                            .Create(s_clientIdForPublicApp)
                            .WithAuthority(GetAuthority())
                            .WithLogging(Log, LogLevel.Verbose, true)
#if !NET70
                            .WithDesktopFeatures()
@#endif
                            .WithExperimentalFeatures();
```

**Review by trwalke — APPROVED** [review]

**Comment by pmaytak** [comment]
Closing, discussed previously to update platforms on existing apps, if needed.

**Pull request closed without merging**